rotkehlchen/exchanges/binance.py

```diff
@@ -530,6 +537,66 @@ def _query_spot_balances(
 
         return balances
 
+    def _query_funding_balances(
+            self,
+            balances: defaultdict[AssetWithOracles, Balance],
+    ) -> defaultdict[AssetWithOracles, Balance]:
+        funding_balances = self.api_query_list('sapi', 'asset/get-funding-asset', request_type='post')  # noqa: E501
```

Comment:
the errors that this function can raise are not handled here. Are they handle somewhere else? If so please point it in the docstring otherwise handle them here